### PR TITLE
Browser changes language automatically

### PR DESCRIPTION
**Pull request by DavDan1**
PT:https://www.pivotaltracker.com/story/show/178557440
___
### User story
```
As a user
In order to see the website in my own browsers language
We need to create language detector to detect the browser language
```
___
### Changes suggested
- Fixing working hours bug
- Add language switching option
- Adds automatic language detection
- Ahoy tracking of submit and phone calls

### CODE REVIEW

**Review by arunbhalli — APPROVED** [review]

File: src/modules/Inquiries.js [hunk 1]
```diff
@@ -8,13 +9,16 @@ const Inquiries = {
...
       let params = { inquiry: formData }
       let response = await axios.post('/inquiries', params)
       store.dispatch({
         type: 'SET_SUBMIT_MESSAGE',
         payload: response.data.message,
-      })
+      })      
+      ahoy.track(`answer`, question: 'submit');
     } catch (error) {}
+
```
Comment (LucasKnudsen) [inline src/modules/Inquiries.js]: `{question: 'submit'}` should be an object :D

File: src/i18n.js [hunk 2]
```diff
@@ -8,15 +8,30 @@ const resources = {
...
-    useSuspense: true,
-  },
-})
+let fallbackLng = undefined
+if (navigator.language.includes('sv')) {
+  fallbackLng = 'Svenska'
+} else {
+  fallbackLng = 'English'
+}
+
```
Comment (LucasKnudsen) [inline src/i18n.js:17]: If short answer exists, what was the reason that you had to change both here and in useEffect?

**Review by LucasKnudsen — CHANGES_REQUESTED** [review]

File: src/i18n.js (hunk 2 above)
Comment (offcast-dev) [reply src/i18n.js:17]: We did it for dropdown, as it wasn't showing correct language. But now as we are changing fallback language we dont' need it anymore. Removed it still works fine

File: src/modules/Inquiries.js (hunk 1 above)
Comment (offcast-dev) [reply src/modules/Inquiries.js]: ok

**Review by LucasKnudsen — APPROVED** [review]

**Pull request merged**